src/runtime/pkg/oci/utils.go

```diff
@@ -556,9 +567,6 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig,
 
 		config.HypervisorConfig.SGXEPCSize = size
 	}
-	if initdata, ok := ocispec.Annotations[vcAnnotations.Initdata]; ok {
-		config.HypervisorConfig.Initdata = initdata
```

Comment:
cc @huoqifeng @mkulke note that I did a little change here https://github.com/kata-containers/kata-containers/pull/10610/commits/21cd3b6c521fc79fec67d450cc64070d03e1ae23#diff-539488d870e5ac4cce9d7f6302c81d9737b4b2bc9e7a404a42b1de82b35d070bR965 This would set the `Initdata` field of `HypervisorConfig` actually the initdata toml plaintext. Idk if it will do any influence to peerpod?

Reply:
yes, if we gzip the body we'll have to adjust the parsing code, I guess this should be fine as long as we merge this not too late in the release cycle

Reply:
yes the extraction logic needs to be adjusted on the CAA side if the body is gzipped now. as long as this is merged not too late it the release cycle it should be an easy fix

Reply:
Hi @mkulke note that this PR has been merged.